modules/collect/src/main/java/com/opengamma/strata/collect/io/CsvOutput.java

```diff
@@ -222,6 +434,22 @@ public void writeLines(Iterable<? extends List<String>> lines, boolean alwaysQuo
     }
   }
 
+  /**
+   * Writes multiple {@code CsvRow}s to the underlying.
+   * <p>
+   * The boolean flag controls whether each entry is always quoted or only quoted when necessary.
+   *
+   * @param rows  the rows to write
+   * @param alwaysQuote  when true, each column will be quoted, when false, quoting is selective
+   * @throws UncheckedIOException if an IO exception occurs
+   */
+  public void writeRows(Iterable<? extends CsvRow> rows, boolean alwaysQuote) {
```

Comment:
* Method should be next to `writeRow` (after `writeLine`)
* `CsvRow` is final, so `? extends` won't add much.

Reply:
Keep writeLine_s_ and writeRow_s_ next to each other?